CDCgov/prime-simplereport
# Elisa/5321 remove supported disease frontend

# FRONTEND PULL REQUEST

## Related Issue
Resolves #5321 

## Changes Proposed
- Removes usages of `DeviceType.SupportedDiseases` from the frontend
- Adds mocks of `supportedDiseaseTestPerformed` to be reused in tests
- Updates storybook

## Additional Information

## Testing
Deployed on `dev2` for testing!
Please ensure the following continue to work as intended:

**Support Admin**
- Create a new device
- Edit a new device

**Facility Settings**
- Add/Remove device to facility

**Test Queue**
- Add a test to the queue and change between a multiplex and covid test
- Ensure timer on test queue continues to work
(Note switching between devices with different time lengths doesn't update the time on the test queue without a refresh -- this seems to be a current bug and not one introduced in this PR -- will create ticket for this)

**Device Code Lookup**
- searching for a device shows the supported disease in the search bar

**storybook**
- ensure storybook for "QueueItem" works as intended
